elements/rh-alert/rh-alert.ts

```diff
@@ -62,13 +76,19 @@ export class RhAlert extends LitElement {
         </div>
         <div id="middle-column">
           <header>
-            <div id="header"><slot name="header"></slot></div>
+            <div id="header">
+              <slot name="header"></slot>
+            </div>
             <div id="header-actions">
-              <button id="close-button" aria-label="Close" confirm>${ICONS.get('close')}</button>
+              <button id="close-button" aria-label="Close" confirm @click=${this._closeHandler}>${ICONS.get('close')}</button>
```

Comment:
That could've been a piece that I had missed in our conversation, though I think that makes sense thinking on it in retrospect since we don't necessarily need it all the times for example on a toast message.